### PR TITLE
Feature/add episodes to story

### PR DESCRIPTION
**Pull request by tonysorensen**
# Description

- What work was done?

Changes were made to storyNewModel.js to return an array of episodes along with the story object. Redundant route /storiesNew/:"{ID}/:episodes was removed from storyNewRouter.js. Note left in test file to create Jest test in the future.

- Why was this work done?

In order to return a more complete story object.

- What feature / user story is it for?

"Add episodes to the story/:id endpoint"

- Any relevant links or images / screenshots
- https://www.loom.com/share/22459a0ea5154fb3be67b1cc70c31991

https://trello.com/c/JTNRnC1Z

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x ] Complete, but not tested (may need new tests)

## Has This Been Tested

- [X] Yes (Tested in Insomnia/Postman, unit tests not written)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: api/stories/storyNewModel.js [hunk 1]
```diff
@@ -17,8 +17,22 @@ const add = (story) => {
...
-  return db('Stories-New').where({ ID });
+const getById = async (ID) => {
+  const story = await db('Stories-New').where('Stories-New.ID', ID);
+  const episodes = await getEpisodesByStoryID(ID);
+  const episodesArray = [];
+  let i = 0;
+  for (i; i < episodes.length; i++) {
+    episodesArray.push(episodes[i]);
+  }
+  const storyWithEpisodes = {
```
Comment (Aaron-TheCreator) [inline api/stories/storyNewModel.js:28]: this is where we did some basic object manipulation to create a JSOn return object that includes the return data from the 'getEpisodesByStoryID' function in an array, assigned to an "Episodes" property.

**Review by Aaron-TheCreator — CHANGES_REQUESTED** [review]
it was awesome working onthis with you, Tony. I'm going to record a Loom video of this working on my machine to add to this PR

File: api/stories/storyNewModel.js (hunk 1 above)
Comment (Aaron-TheCreator) [reply api/stories/storyNewModel.js:28]: Loom video added, convo resolved

**Review by Aaron-TheCreator — APPROVED** [review]

**Review by chukalicious — APPROVED** [review]
Tested on Postman and the request returned a story object that contains an episodes array for the selected story. Great work, guys!

File: api/stories/storyNewModel.js [hunk 2]
```diff
@@ -17,8 +17,22 @@ const add = (story) => {
...
  * @returns {Promise} a promise that resolves to story object of the given story ID
  */
-const getById = (ID) => {
-  return db('Stories-New').where({ ID });
+const getById = async (ID) => {
+  const story = await db('Stories-New').where('Stories-New.ID', ID);
+  const episodes = await getEpisodesByStoryID(ID);
+  const episodesArray = [];
+  let i = 0;
+  for (i; i < episodes.length; i++) {
```
Comment (ryan-hamblin) [inline api/stories/storyNewModel.js:25]: Why are we choosing to scope that index var `i` above the for loop outside of the loop's parens.

Comment (Aaron-TheCreator) [reply api/stories/storyNewModel.js:25]: Great question! There is not a good answer. This is fixed on branch "feature/add-prompts-to-story" which I will PR tonight.  #78

**Pull request merged**